src/RA_Defs.h

```diff
@@ -286,4 +296,18 @@ typedef std::basic_string<TCHAR> tstring;
 #define NativeStrType std::string
 #endif
 
+namespace ra {
+
+// temp, something similar is in the GSL but need to fix up the Memory Bookmarker - Samer
+// Didn't include my type_traits here for fear people would be scared of lots of lines
+template<typename Container>
+_NODISCARD auto at(typename Container::size_type pos, Container& myContainer) noexcept
+->decltype(std::declval<Container&>().at(pos))
+{
+    return myContainer.at(pos);
+}
+
+} // namespace ra 
```

Comment:
This was used for the MemBookmark dialog since there wasn't a getter for the bookmark map.